release_tester/arangodb/installers/__init__.py

```diff
@@ -214,7 +214,8 @@ def __init__(
 
         self.base_test_dir = test_dir
         self.pwd = Path(os.path.dirname(os.path.realpath(__file__)))
-        self.test_data_dir = self.pwd / ".." / ".." / ".." / "test_data"
+        self.test_data_dir = self.pwd / ".." / ".." / ".." / "rta_makedata" / "test_data"
```

Comment:
https://jenkins01.arangodb.biz/view/RTA/job/Test-3.10.Linux/242/allure/#suites/0a4f68960c8f77a85053925f9ae1d6b5/35cfefb916b6090e/

We get an exception here. It goes away if I change the path to "rta-makedata" here.
Please clarify whether having 2 directories(rta_makedata and rta-makedata) was intentional or not.